### PR TITLE
NIFI-9404 Add NAR info, parent NAR info, and build info to extension-…

### PR DESCRIPTION
**Pull request by bbende**
…manifest

### CODE REVIEW

File: src/main/java/org/apache/nifi/NarMojo.java [hunk 1]
```diff
@@ -550,11 +551,44 @@ private void generateDocumentation() throws MojoExecutionException {
             try {
                 xmlWriter.writeStartElement("extensionManifest");
```
Comment (ottobackwards) [inline src/main/java/org/apache/nifi/NarMojo.java:553]: Should there be some author Information in here?

**Comment by ottobackwards** [comment]
This looks pretty straight forward, I have two questions though:

1.  should there be github actions to build and lint this?
2. How is this Information used be the registry?  is it visible to the operator?   Should there be information about the author included as well?

File: src/main/java/org/apache/nifi/NarMojo.java (hunk 1 above)
Comment (bbende) [reply src/main/java/org/apache/nifi/NarMojo.java:559]: If the definition of "author" is the person that wrote the code, then no I don't think we have a way to really track this, but I did push an update that adds "builtBy" which is consistent with what we have in the MANIFEST.

**Comment by bbende** [comment]
> should there be github actions to build and lint this?

It was never setup for this repo since it is very low activity. I'd say it would be a follow on effort if we want to do that.

> How is this Information used be the registry? is it visible to the operator? Should there be information about the author included as well?

The registry currently always has the NAR so it can obtain all this info from the MANIFEST, but I want to be able to have the same info available when only having the XML file and not having the full NAR.

**Comment by joewitt** [comment]
+1 merged to main- thanks

**Comment by ottobackwards** [comment]
Sorry, I wasn't able to check my mail on this before now, thanks for answering @bbende , and thanks for doing the merge @joewitt

**Pull request closed without merging**